pcrs/pcr4.md

```diff
@@ -0,0 +1,20 @@
+# PCR4
+
+It measures the boot process, including attempted boot devices, and boot
+code loaded and executed from the device.
+
+
+## Events
+- `EV_EFI_ACTION`: sha256 digest of the `Calling EFI Application from
+  Boot Option` string.
+- `EV_SEPARATOR`: Value `0000000` measured prior to the invocation of
+  the first Ready to Boot call.
+- `EV_EFI_BOOT_SERVICES_APPLICATION`: Hash digest of the shim binary.
+- `EV_EFI_BOOT_SERVICES_APPLICATION`: Hash digest of the grub binary.
+- `EV_EFI_BOOT_SERVICES_APPLICATION`: Hash digest of vmlinuz.
```

Comment:
It's the authenticode hash in this case. See: https://github.com/rhboot/shim/blob/main/README.tpm#L3